requirements.txt

```diff
@@ -4,9 +4,9 @@ jsonschema
 matplotlib
 netcdf4
 cfgrib
-ecmwflibs
-numpy>=1.21.6,<=1.24.1
-pandas>=1.3.5,<=1.4.4
+#ecmwflibs
```

Comment:
Might make sense to also comment out cfgrib here since it is used for the same thing as ecmwflibs and including it on its own generates a warning (or used to for me).